### ISSUE TITLE
FixMatch with EMA ?

### ISSUE THREAD

**Issue by willer94** [microsoft/Semi-supervised-learning#9]
Hi, thanks for the code base. However, it seems that ema has not been integrated  into FixMatch, which is different to the implementation in the original paper.

**Comment by Hhhhhhao** [comment]
EMA is enabled for all classic cv settings as: https://github.com/microsoft/Semi-supervised-learning/blob/d994927b6bc3ad902fb220626489610e69a53fd0/config/classic_cv/fixmatch/fixmatch_cifar10_40_0.yaml#L19

However, for settings in USB, we do not use EMA since we find there is no significant difference of using it or not with the pre-trained models.

**Comment by willer94** [comment]
Thanks for the quick reply! I thought that the logits of weak augmented samples were predicted by the ema model. I have checked the original implementation in TF and found that I got the wrong idea.
